### PR TITLE
UIWebView was removed

### PR DESCRIPTION
**Pull request by yoalex5**
>ITMS-90809: Deprecated API Usage - Apple will stop accepting submissions of apps that use UIWebView APIs . See https://developer.apple.com/documentation/uikit/uiwebview for more information.

1. `UIWebView` was removed from SDK
2. `WKWebView` was set by default for AM SDK

**AdManager**
[AdManager Release notes](https://developers.google.com/ad-manager/mobile-ads-sdk/ios/rel-notes)
> 7.55.0 | 2020‑02‑04 | Removed all references to UIWebView. UIWebView is no longer supported.

**MoPub**
[MoPub issue](https://github.com/mopub/mopub-ios-sdk/issues/285)
[MoPub Release notes](https://developers.mopub.com/publishers/ios/changelog/)
>Version 5.9.0 (September 16, 2019) Totally remove UIWebView

`AdViewUtils.findPrebidCreativeSize` works only with `WKWebView`.
`AdManager`: `gad_preferred_webview` key with `wkwebview` value or v7.55.0(or higher)
```
//Info.plist 
<key>gad_preferred_webview</key>
<string>wkwebview</string>
```
`MoPub`: v5.9.0(or higher)

### CODE REVIEW

File: Example/PrebidDemo/PrebidDemoObjectiveC/Info.plist [hunk 1]
```diff
@@ -41,14 +41,18 @@
...
 	<array>
 		<string>UIInterfaceOrientationPortrait</string>
 		<string>UIInterfaceOrientationPortraitUpsideDown</string>
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
+
+    <key>GADIsAdManagerApp</key>
+    <true/>
+    <key>gad_preferred_webview</key>
```
Comment (ppuviarasu) [inline Example/PrebidDemo/PrebidDemoObjectiveC/Info.plist]: Do we need to set this even if UIWebView is removed completely from the SDK?

File: Source/Addendum/AdViewUtils.swift [hunk 2]
```diff
@@ -27,19 +27,16 @@ public final class AdViewUtils: NSObject {
     @objc
     public static func findPrebidCreativeSize(_ adView: UIView, success: @escaping (CGSize) -> Void, failure: @escaping (Error) -> Void) {
         
-        let view = self.findWebView(adView) { (subView) -> Bool in
-            return isWebView(subView)
```
Comment (ppuviarasu) [inline Source/Addendum/AdViewUtils.swift]: I think this is still relevant. It should be isWebView irrestpective of WKWebView or UIWebView...

File: Source/Addendum/AdViewUtils.swift [hunk 3]
```diff
@@ -52,22 +49,22 @@ public final class AdViewUtils: NSObject {
         failure(error)
     }
     
-    static func findWebView(_ view: UIView, closure:(UIView) -> Bool) -> UIView? {
```
Comment (ppuviarasu) [inline Source/Addendum/AdViewUtils.swift]: Same... we are finding recursively for WebView so the naming is apt... lets keep it as is

File: Source/Addendum/AdViewUtils.swift [hunk 4]
```diff
@@ -147,8 +128,8 @@ public final class AdViewUtils: NSObject {
         return matchAndCheck(regex: AdViewUtils.sizeValueRegexExpression, text: hbSizeObject)
     }
     
-    static func isWebView(_ view: UIView) -> Bool {
```
Comment (ppuviarasu) [inline Source/Addendum/AdViewUtils.swift]: Let the naming remain same

File: Example/PrebidDemo/PrebidDemoObjectiveC/Info.plist (hunk 1 above)
Comment (yoalex5) [reply Example/PrebidDemo/PrebidDemoObjectiveC/Info.plist]: This case is useful when AM SDK has a version that is lower then `v7.55.0` where `UIWebView` is used by default

File: Source/Addendum/AdViewUtils.swift (hunk 2 above)
Comment (yoalex5) [reply Source/Addendum/AdViewUtils.swift]: In my opinion `WebView` is a common definition which covers the whole set of classes(`UIWebView`, `WKWebView`) with corresponded functionality. 

Theoretically `UIWebView` can be passed into it and success result is expected. That is why I decided to specify a concrete type(`WKWebView`) for this logic

File: Source/Addendum/AdViewUtils.swift (hunk 3 above)
Comment (yoalex5) [reply Source/Addendum/AdViewUtils.swift]: This function can be used for different purposes 

For example:
``` Swift
let view = self.findWebView(myView) { (subView) -> Bool in
    return subView is UIButton
}
```

File: Source/Addendum/AdViewUtils.swift (hunk 4 above)
Comment (yoalex5) [reply Source/Addendum/AdViewUtils.swift]: `isWKWebView` has more clear naming

**Comment by yoalex5** [comment]
@ppuviarasu, could you please review my comments ?

**Pull request merged**